swarm/tracing/tracing.go

```diff
@@ -85,13 +84,13 @@ func initTracer(endpoint, svc string) (closer io.Closer) {
 	// Example logger and metrics factory. Use github.com/uber/jaeger-client-go/log
 	// and github.com/uber/jaeger-lib/metrics respectively to bind to real logging and metrics
 	// frameworks.
-	jLogger := jaegerlog.StdLogger
+	//jLogger := jaegerlog.StdLogger
```

Comment:
Remove these loggings?

Reply:
I reverted these changes in d9019c8bbac7cb91d12cbde7d043c274943956a5